lib/rgeo/coord_sys/cs/wkt_parser.rb

```diff
@@ -79,6 +79,13 @@ def parse(containing_type = nil) # :nodoc:
             obj = VerticalDatum.create(args.shift(QuotedString), args.shift(Numeric), *args.create_optionals)
           when "LOCAL_DATUM"
             obj = LocalDatum.create(args.shift(QuotedString), args.shift(Numeric), *args.create_optionals)
+          when "CS"
```

Comment:
The only issue is: what happens when we parse wkt using geos and its default implementation ?

Reply:
Would the geos parser try to parse the CS stuff as well? I can check again, but I thought this parser is only used in `CS::*` modules and wouldn't be used when a geos factory is attempting to parse WKT geometries for example.

Reply:
Oh maybe not, you are right